devel/nodejs10/Portfile

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+PortGroup               compiler_blacklist_versions 1.0
```

Comment:
Do you need to add the PG ? You don’t specify any blacklist later on.

Reply:
There is one already in the portfile later on, and he has this PG in all the other nodejs Portfiles.

In fact I thought that the cxx11 PG would be enough, and would call in the compiler_blacklist_versions PG so you wouldn't need to include it separately, but I've seen situations where I saw errors when I didn't include it (probably a libc++ / libstdc++ thing).

Reply:
Here's an example. The nodej8 Portfile in this PR includes both the compiler_blacklist_versions PG, and shortly thereafter the cxx11 1.1 PG. And I see:
```
$ port lint
--->  Verifying Portfile for nodejs8
--->  0 errors and 0 warnings found.
```
but if I remove the (should be redundant) compiler_blacklist_versions PG, I get:

```
$ port lint
--->  Verifying Portfile for nodejs8
Error: Line 106 uses compiler.blacklist entry {clang < 500} which requires the compiler_blacklist_versions portgroup which has not been included
--->  1 errors and 0 warnings found.
```
And yet, the cxx11 1.1 PG includes the compiler_blacklist_versions PG as it's first step. So -- I don't understand, exactly, why this happens...

Reply:
perhaps `port lint` doesn't understand when a PG includes another one ?

Reply:
Indeed, `port lint` doesn't understand when a PG includes another one. @ryandesign pointed out somewhere (I forgot where) that's better to explicitly list all used PGs and don't rely on the fact that on PG includes another. So `port lint` is right in this regard.